File: frontend/src/app/model/project.ts

export class Project {
    public id: number;
    public name: string;
    public standard: boolean;
    public user: string;


    constructor(id: number, name: string, standard: boolean, user?: string) {
        this.id = id;
        this.name = name;
        this.standard = standard;
        this.user = user;
    }
}

File: backend/src/main/java/ch/zhaw/sml/iwi/meng/leantodo/entity/Project.java
package ch.zhaw.sml.iwi.meng.leantodo.entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Project {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String title;

    private Boolean standard = false;

    private String user;


    // GETTER AND SETTER
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public Boolean isStandard() {
        return standard;
    }

    public void setStandard(Boolean standard) {
        this.standard = standard;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }
}
